mjpc/tasks/humanoid/tracking/task.cc

```diff
@@ -37,6 +38,55 @@ std::tuple<int, int, double, double> ComputeInterpolationValues(double index,
 
   return {index_0, index_1, weight_0, weight_1};
 }
+
+// return length of motion trajectory
+int TrajectoryLength(int id) {
+  // Jump
+  if (id == 0) {
+    return 121;
+  // Kick Spin
+  } else if (id == 1) {
+    return 154;
+  // Spin Kick
+  } else if (id == 2) {
+    return 115;
+  // Cartwheel (1)
+  } else if (id == 3) {
+    return 78;
+  // Crouch Flip
+  } else if (id == 4) {
+    return 145;
+  // Cartwheel (2)
+  } else if (id == 5) {
+    return 188;
+  // Monkey Flip
+  } else if (id == 6) {
+    return 260;
+  // Dance
+  } else if (id == 7) {
+    return 279;
+  // Run
+  } else if (id == 8) {
+    return 39;
+  // Walk
+  } else if (id == 9) {
+    return 510;
+  // Stand
+  } else if (id == 10) {
+    return 1;
```

Comment:
Is this one missing the actual keyframe?

Reply:
Removed.